webknossos/webknossos/administration/task.py

```diff
@@ -33,13 +38,27 @@ def open_instance_count(self) -> int:
         return self.pending_instance_count
 
 
+@attr.frozen
+class TaskExperience:
+    """Data class containing information about the experience needed to work on a task"""
+
+    domain: str
+    value: int
+
+    @classmethod
+    def _from_api_experience(cls, api_experience: ApiExperience) -> "TaskExperience":
+        return cls(api_experience.domain, api_experience.value)
+
+
 @attr.frozen
 class TaskType:
```

Comment:
Please ask the AI to add a small Example to each doc string for all (static) methods.